WcaOnRails/app/controllers/delegates_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class DelegatesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Yes, that happens at https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/controllers/competitions_controller.rb#L6-L12 and https://github.com/cubing/worldcubeassociation.org/blob/72da7bf81043eee8692e418ab2383809e3ba4b1e/WcaOnRails/app/controllers/posts_controller.rb#L3